<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetectorWindow</class>
 <widget class="QMainWindow" name="DetectorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1285</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_EIF">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1265</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Engineering Inspection Function</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 127);</string>
    </property>
    <property name="text">
     <string>Input Path</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_inputfolder">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>50</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_output">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>50</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 127);</string>
    </property>
    <property name="text">
     <string>Output Path</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_outputfolder">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_weightfile">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>50</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 127);</string>
    </property>
    <property name="text">
     <string>Weight Path</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_weightfile">
    <property name="geometry">
     <rect>
      <x>945</x>
      <y>50</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_detect">
    <property name="geometry">
     <rect>
      <x>1155</x>
      <y>50</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>Detection</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>620</width>
      <height>420</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Input Image Visualization</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_output">
    <property name="geometry">
     <rect>
      <x>655</x>
      <y>120</y>
      <width>620</width>
      <height>420</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Prediction Image Visualization</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
